xline/src/server/lease_server.rs

```diff
@@ -41,10 +40,10 @@ where
     client: Arc<Client<Command>>,
     /// Server name
     name: String,
-    /// State of current node
-    state: Arc<State>,
     /// Id generator
     id_gen: Arc<IdGenerator>,
+    /// Address of all members
+    all_members: HashMap<String, String>,
```

Comment:
I believe it's not the only server that knows the address of all members. Shall we share the info?

Reply:
Whether a server need to know the address of all members or not depends on whether it needs to forward its requests to the leader. And currently, only the lease server depends on it.